<ui version="4.0" >
 <class>CustomDialog</class>
 <widget class="QDialog" name="CustomDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Customize</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page" >
      <attribute name="title" >
       <string>Preference</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="../images/application.qrc" >:/desktop.png</iconset>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Application preference</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="4" column="1" >
           <widget class="QComboBox" name="m_langComboBox" >
            <item>
             <property name="text" >
              <string>(en) English</string>
             </property>
             <property name="icon" >
              <iconset resource="../images/application.qrc" >:/usa.png</iconset>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>(fr_FR) French</string>
             </property>
             <property name="icon" >
              <iconset resource="../images/application.qrc" >:/france.png</iconset>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>XINX Lang :</string>
            </property>
            <property name="buddy" >
             <cstring>m_langComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2" >
           <widget class="QCheckBox" name="m_popupWhenFileModifiedCheckBox" >
            <property name="text" >
             <string>&amp;Popup when file modifed</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2" >
           <widget class="QCheckBox" name="m_saveSessionCheckBox" >
            <property name="text" >
             <string>Save session by default</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2" >
           <widget class="QCheckBox" name="m_createBakCheckBox" >
            <property name="text" >
             <string>Create backup file (.bak)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2" >
           <widget class="QCheckBox" name="m_alertStandardCheckBox" >
            <property name="text" >
             <string>Alert user when file is standard (project mode only)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <attribute name="title" >
       <string>Path</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="../images/application.qrc" >:/folder.png</iconset>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Custom path</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="3" >
           <widget class="QToolButton" name="m_changeObjectPathBtn" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QLineEdit" name="m_objectDescriptionPathLineEdit" />
          </item>
          <item row="0" column="3" >
           <widget class="QToolButton" name="m_changeProjectPathBtn" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>XINX project directory :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2" >
           <widget class="QLineEdit" name="m_projectPathLineEdit" />
          </item>
          <item row="1" column="0" colspan="2" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Business object description :</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Project files path</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Path :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="m_fileTypePath" />
          </item>
          <item row="0" column="0" colspan="2" >
           <widget class="QComboBox" name="m_fileTypeComboBox" >
            <item>
             <property name="text" >
              <string>Stylesheet (*.xsl)</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>JavaScript (*.js)</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Configuration file (configuration.xml)</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0" colspan="2" >
           <widget class="QCheckBox" name="m_specifiqueCheckBox" >
            <property name="text" >
             <string>Can be specifique</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>Tools</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_7" >
              <property name="text" >
               <string>CVS : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_CVSToolsLineEdit" />
            </item>
            <item>
             <widget class="QToolButton" name="m_cvsToolButton" >
              <property name="text" >
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3" >
      <attribute name="title" >
       <string>Editor</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="../images/application.qrc" >:/style.png</iconset>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Customize syntax</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="4" column="0" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Font :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3" >
           <widget class="QCheckBox" name="m_underlineCheckBox" >
            <property name="font" >
             <font>
              <underline>true</underline>
             </font>
            </property>
            <property name="text" >
             <string>Underline</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2" >
           <widget class="QComboBox" name="m_syntaxTypeComboBox" />
          </item>
          <item row="0" column="0" colspan="2" >
           <widget class="QComboBox" name="m_syntaxFileTypeComboBox" />
          </item>
          <item row="2" column="3" >
           <widget class="QCheckBox" name="m_StrikeoutCheckBox" >
            <property name="font" >
             <font>
              <strikeout>true</strikeout>
             </font>
            </property>
            <property name="text" >
             <string>Strikeout</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" >
           <widget class="QCheckBox" name="m_italicCheckBox" >
            <property name="font" >
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text" >
             <string>Italic</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3" >
           <widget class="QCheckBox" name="m_boldCheckBox" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>Bold</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Color :</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="3" >
           <widget class="QFontComboBox" name="m_fontComboBox" />
          </item>
          <item row="3" column="1" colspan="3" >
           <widget class="KColorCombo" name="m_colorComboBox" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="m_exempleTextEdit" >
         <property name="lineWrapMode" >
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
         <property name="tabStopWidth" >
          <number>15</number>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>CVS</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="../images/application.qrc" >:/cervisia.png</iconset>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="1" column="1" >
          <widget class="QComboBox" name="m_cvsCompressionComboBox" >
           <item>
            <property name="text" >
             <string>-z0 Disabled </string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>-z1 Low level</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>-z2</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>-z3</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>-z4</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>-z5 Medium Level</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>-z6</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>-z7</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>-z8</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>-z9 High level</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_9" >
           <property name="text" >
            <string>Compression:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2" >
          <widget class="QCheckBox" name="m_cvsPruneCheckBox" >
           <property name="text" >
            <string>Prune empty directories. (-P)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="m_cvsVerboseComboBox" >
           <item>
            <property name="text" >
             <string>Very Quiet (-Q)</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Discret (-q)</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Verbose</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0" colspan="2" >
          <widget class="QCheckBox" name="m_cvsCreateDirCheckBox" >
           <property name="text" >
            <string>Create any directories that exist in the repository if they're missing from the working directory. (-d)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="label_8" >
           <property name="text" >
            <string>Progress message: </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="icon" >
        <iconset resource="../images/application.qrc" >:/button_ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="../images/application.qrc" >:/button_cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>../src/kcolorcombo.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/application.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>CustomDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>296</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CustomDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>379</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
